backends/p4tools/testgen/targets/bmv2/register.h

```diff
@@ -0,0 +1,23 @@
+#ifndef TESTGEN_TARGETS_BMV2_REGISTER_H_
+#define TESTGEN_TARGETS_BMV2_REGISTER_H_
+
+#include "backends/p4tools/common/p4ctool.h"
+
+#include "backends/p4tools/testgen/options.h"
+#include "backends/p4tools/testgen/targets/bmv2/bmv2.h"
+#include "backends/p4tools/testgen/targets/bmv2/target.h"
+#include "backends/p4tools/testgen/testgen.h"
+
+namespace P4Tools {
+
+namespace P4Testgen {
+
+void bmv2_registerCompilerTarget() { Bmv2::BMv2_V1ModelCompilerTarget::make(); }
```

Comment:
In general I would recommend a little comment in each file describing its purpose.
For example, here based on the file name I thought this handles v1model Registers. 
It seems to be something quite different.

Reply:
In general, we try to add comments to each function definition but sometimes things slip through. I am hoping that we are able to refactor this whole structure very soon to make it a bit simpler. It has grown over time and is starting to become confusing.